examples/great3/rgc.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2012-2015 by the GalSim developers team on GitHub
+# https://github.com/GalSim-developers
+#
+# This file is part of GalSim: The modular galaxy image simulation toolkit.
+# https://github.com/GalSim-developers/GalSim
+#
+# GalSim is free software: redistribution and use in source and binary forms,
+# with or without modification, are permitted provided that the following
+# conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions, and the disclaimer given in the accompanying LICENSE
+#    file.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions, and the disclaimer given in the documentation
+#    and/or other materials provided with the distribution.
+
+# The Great3 real-ground-constant branch configuration file
+# Note: This configuration will not produce images _identical_ to the ones used for Great3, 
+# since the order of random number generation is different.  However, it is intended to
+# produce an _equivalent_ simulation as the one produced for Great3.
+
+# The real-galaxy branch is almost identical to the control branch.  So rather than duplicate
+# everything from cgc.yaml and change a couple items, we use the template feature of GalSim
+# config.
+
+# This copies over everything from cgc.yaml
+template : cgc.yaml
+
+# Now we can change the things we need to change:
+
+# 1. Use real galaxies rather than parametric
+gal.first.gal_type : real
```

Comment:
Yes.  The YAML parser gets you to `config['gal.first.gal_type'] = 'real'`.  Then I have to parse it to convert it to `config['gal']['first']['gal_type'] = 'real'`.